src/main/CommandHandler.cpp

```diff
@@ -213,6 +213,16 @@ parseOptionalParamOrDefault(std::map<std::string, std::string> const& map,
     }
 }
 
+template <>
+bool
+parseOptionalParamOrDefault<bool>(std::map<std::string, std::string> const& map,
+                                  std::string const& key,
+                                  bool const& defaultValue)
+{
+    auto paramStr = parseOptionalParam<std::string>(map, key);
+    return paramStr && *paramStr == "true";
```

Comment:
this condition doesn't respect the default value (if option is not provided, it should return the default)

Reply:
Nice catch, fixed.